### PR TITLE
posts: add events series post

### PR DESCRIPTION
**Pull request by jalehman**
The Urbit Events Series blog post, as laid out in [this doc](https://docs.google.com/document/d/11e_bW7EykJD0BpvZRImAa4nWHlP-07BZa3cQu800fK8/edit). 

There are still a few todos here for @lrettig and @urcades, left as inline TODOs in the markdown.

### CODE REVIEW

File: content/blog/events-series.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+<img class="ba" src="TODO(ED)">
+
+<br>
+
+*Urbit is launching a series of community events, the first of which is a developer call taking place on Thursday, November 5 at 9am PT. Read on for all of the details.*
+
+--
+
+If you’re familiar with the project, you’ve probably noticed that Urbit folks talk a lot about the fact that [Urbit is for communities](https://urbit.org/blog/urbit-is-for-communities/). This is central to what Urbiters believe: Urbit is a platform that any group of people should be able to use to build, communicate, and transact in precisely the way that makes sense to them. Today, Urbit is already capable of facilitating many of these activities, and it’s already a big part of the daily workflow for many people. I think we’re off to a pretty good start, and over time Urbit will become accessible to more communities.
```
Comment (matildepark) [inline content/blog/events-series.md]: > This is central to what Urbiters believe

We don't really have a concrete noun. People just say Martians usually. Just flagging that I haven't seen Urbiter before ("orbiter" joke?)

**Review by matildepark — COMMENTED** [review]
Reads fine, looks fine, pending the TODOs being filled.

File: content/blog/events-series.md (hunk 1 above)
Comment (tylershuster) [reply content/blog/events-series.md]: I like Urbiteer

Comment (lrettig) [reply content/blog/events-series.md]: meant to ask about this! I felt weird saying "Urbit folks" a bunch. I defer to you guys here!

**Pull request merged**